cassandra/src/main/java/com/yahoo/ycsb/db/CassandraCQLClient.java

```diff
@@ -163,6 +166,8 @@ public void init() throws DBException {
 
         String keyspace = getProperties().getProperty(KEYSPACE_PROPERTY,
             KEYSPACE_PROPERTY_DEFAULT);
+        String policyname = getProperties().getProperty(LOAD_BALANCING_POLICY_PROPERTY,
+            LOAD_BALANCING_POLICY_PROPERTY_DEFAULT);
```

Comment:
Making the load-balancing policy as a configurable parameter is good. But I am unsure if DC-aware round-robin policy alone should be the default. Apache Cassandra's own default is datacenter-aware + token-aware policy together.
https://docs.datastax.com/en/developer/java-driver/3.5/manual/load_balancing/. Let's not have a default here.

Reply:
Thank you for your comment. I've updated the PR.